src/adapters/poi/idunn_poi.js

```diff
@@ -46,6 +46,14 @@ export default class IdunnPoi extends Poi {
     case 'address':
     case 'street':
       return this.alternativeName;
+    case 'admin':
+      if (this.address && this.address.label) {
```

Comment:
Using the new optionnal fields syntax, you could replace all these lines with:
```
return this.address?.label?.split(',')[0] || this.name;
```

Reply:
Just because I could doesn't mean I should :D 
(but I guess it's better indeed)

Reply:
Yep, up to you, that's why I validated the PR anyway :)